tests/test_edgeql_ddl.py

```diff
@@ -6369,7 +6371,7 @@ async def test_edgeql_ddl_role_02(self):
             """,
             [{
                 'name': 'foo2',
-                'superuser': True,
+                'superuser': caps & BackendCapabilities.SUPERUSER_ACCESS,
```

Comment:
No, `superuser` does not actually reflect `rolsuper` directly from Postgres, but rather membership in the EDGEDB_SUPERGROUP role.  However, it appears that we're forgetting to make new roles members of it in `CreateRole`, which is why you get this failure.  IOW, fix `CreateRole.apply` in `pgsql/delta.py` to inherit from `EDGEDB_SUPERGROUP` (unconditionally) instead of from `instance_params.base_superuser` directly.